ballerina/types.bal

```diff
@@ -239,9 +239,9 @@ public enum EntityType {
 # Message receiver modes
 public enum ReceiveMode {
     @display {label: "RECEIVE AND DELETE"}
-    RECEIVE_AND_DELETE = "RECEIVEANDDELETE",
+    RECEIVE_AND_DELETE = "RECEIVE_AND_DELETE",
     @display {label: "PEEK LOCK"}
-    PEEK_LOCK = "PEEKLOCK"
+    PEEK_LOCK = "PEEK_LOCK"
 }
```

Comment:
Here can't we just define the enum without values as below? 

```ballerina
public enum ReceiveMode {
    RECEIVE_AND_DELETE,
    PEEK_LOCK
}

Reply:
Fixed with [9cbb8d050a025b41c9ed6372405da48e997cc42c](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/220/commits/9cbb8d050a025b41c9ed6372405da48e997cc42c)